google/cloud/bigtable/options.h

```diff
@@ -51,6 +51,15 @@ namespace cloud {
 namespace bigtable {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
+/**
+ * The application profile id needed for using the replication API.
```

Comment:
"Option" and "needed" sound like opposites.

Reply:
Lol. It is _needed_ to use an _optional_ feature. I updated the comment to avoid this confusion.

(not related to your comment, but) I also added some more information on app profiles.